lisa/sut_orchestrator/azure/features.py

```diff
@@ -1601,3 +1611,93 @@ def create_setting(
             return nvme
 
         return None
+
+
+class Nfs(AzureFeatureMixin, features.Nfs):
+    @classmethod
+    def create_setting(
+        cls, *args: Any, **kwargs: Any
+    ) -> Optional[schema.FeatureSettings]:
+        return schema.FeatureSettings.create(cls.name())
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        super()._initialize(*args, **kwargs)
+        self._initialize_information(self._node)
+
+    def create(self) -> str:
+        platform: AzurePlatform = self._platform  # type: ignore
+        node_context = self._node.capability.get_extended_runbook(AzureNodeSchema)
+        location = node_context.location
+        resource_group_name = self._resource_group_name
+        random_str = "".join(
+            random.choices(string.ascii_lowercase + string.digits, k=10)
```

Comment:
1. does it work, if it starts with digit?
2. `util/__init__.py` has an implementation: `generate_random_chars`, you can change it to use `choices`

Reply:
updated